src/main/java/org/csanchez/jenkins/plugins/kubernetes/GarbageCollection.java

```diff
@@ -149,13 +165,14 @@ protected void execute(TaskListener listener) throws IOException, InterruptedExc
                                         if (lastRefresh != null) {
                                             try {
                                                 var refreshTime = Long.parseLong(lastRefresh);
+                                                var now = Instant.now();
                                                 LOGGER.log(
-                                                        Level.FINEST,
-                                                        () -> getQualifiedName(pod) + " last refreshed at "
-                                                                + refreshTime);
-                                                return Duration.between(
-                                                                        Instant.ofEpochMilli(refreshTime),
-                                                                        Instant.now())
+                                                        Level.FINE,
```

Comment:
BTW you can use `Logger.fine` with a `Supplier` which is a bit more concise. (Not, alas, with a `Throwable`.)

Reply:
> Not, alas, with a `Throwable`

Indeed, this is why I use this form for consistency.